bitnami/keycloak/Chart.yaml

```
@@ -33,4 +33,4 @@ maintainers:
 name: keycloak
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/keycloak
-version: 18.0.2
+version: 18.0.3
```

As we are adding a new feature, we should update the minor version of the chart:

```suggestion
version: 18.1.0
```